docs/source/development/coding_conventions.rst

```diff
@@ -231,6 +231,56 @@ corresponds to complexity of the body).
 
 |x| **Do not** declare imports not used within the file.
 
+Element ID
+----------
```

Comment:
@nickdotreid I am not sure about that, there is Introduction with Goals and Guideline Presentation at the beginning of document that explains why and how to use it so adding more info about that later in doc feels a bit redundant. I will ask Sebastian what he thinks about it.